src/oscar/apps/checkout/mixins.py

```diff
@@ -146,7 +146,9 @@ def place_order(self, order_number, user, basket, shipping_address,
             shipping_charge=shipping_charge,
             total=order_total,
             billing_address=billing_address,
-            status=status, **kwargs)
+            status=status,
+            request=getattr(self, 'request', None),
```

Comment:
This will break things is someone has passed a `request` to `kwargs`? Need to pop it from `kwargs` if it exists.